Web of Science.js

```diff
@@ -0,0 +1,211 @@
+{
+	"translatorID": "88e11bcb-464d-4b6d-a446-8994e3b865c9",
+	"label": "Web of Science",
```

Comment:
AFAIK that is the same and as far as I can tell Web of Science is more used nowadays. See http://en.wikipedia.org/wiki/Web_of_Science or http://rzblx10.uni-regensburg.de/dbinfo/detail.php?bib_id=alle&colors=&ocolors=&lett=fs&tid=0&titel_id=3366